lib/fizzy/execute.cpp

```diff
@@ -19,6 +20,11 @@ namespace
 // code_offset + imm_offset + stack_height
 constexpr auto BranchImmediateSize = 3 * sizeof(uint32_t);
 
+constexpr auto ABS_MASK = 0x7fffffff;
+constexpr auto SIGN_MASK = 0x80000000;
```

Comment:
This can be based on the other mask.
```suggestion
constexpr uint32_t F32SignMask = ~F32AbsMask;
```